R/classifiers.R

```diff
@@ -1,9 +1,27 @@
-#' Unweighted kTSP classifier
+#' Unweighted k-Top Scoring Pairs (kTSP) classifier
+#'
+#' This unweighted kTSP classifier was trained using `multiclassPairs::train_RF()` to predict the subtype of a given Medulloblastoma sample.
```

Comment:
Wrong function for kTSP:
```suggestion
#' This unweighted kTSP classifier was trained using `multiclassPairs::train_one_vs_rest_TSP()` to predict the subtype of a given Medulloblastoma sample.
```

Reply:
I would also list the possible classes it can predict.